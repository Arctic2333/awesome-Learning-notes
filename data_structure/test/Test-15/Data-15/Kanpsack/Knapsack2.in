6  15.0
¼×  2.5  10.0
ÒÒ  2.0  2.0
±û  30.0  10.0
¶¡  1.0  1.0
Îì  10.0  5.0
¼º  6.5  8.0
