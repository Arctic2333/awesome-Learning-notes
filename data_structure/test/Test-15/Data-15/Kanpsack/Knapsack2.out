±û: 10Kg
Îì: 5Kg
Total price: 40
